sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SparkPlanGraph.scala

```diff
@@ -198,7 +203,7 @@ private[ui] class SparkPlanGraphCluster(
       require(duration.length == 1)
       val id = duration(0).accumulatorId
       if (metricsValue.contains(duration(0).accumulatorId)) {
```

Comment:
While you are here, can we change like this?
```scala
- if (metricsValue.contains(duration(0).accumulatorId)) {
+ if (metricsValue.contains(id)) {
```

Reply:
Sure. I'll do it.